apps/meteor/ee/app/livechat-enterprise/server/settings.ts

```diff
@@ -157,6 +157,32 @@ export const createSettings = async (): Promise<void> => {
 		});
 	});
 
+	await settingsRegistry.add('Livechat_AdditionalWidgetScripts', '', {
+		type: 'string',
+		group: 'Omnichannel',
+		section: 'Livechat',
+		enterprise: true,
+		invalidValue: "",
+		multiline: true,
+		i18nLabel: 'Livechat_AdditionalWidgetScripts',
+		i18nDescription: 'Additional_script_to_embed_in_the_livechat_widget',
```

Comment:
```suggestion
		i18nLabel: 'Livechat_additional_widget_scripts',
		i18nDescription: 'Livechat_additional_widget_scripts_Description',
```

This way you will be able to display the description as a subtext on the field, like this

<img width="594" alt="Screenshot 2023-12-20 at 14 53 11" src="https://github.com/RocketChat/Rocket.Chat/assets/20868078/61bd8f12-2dd7-4137-90f8-c2e8900139db">

The same should be done for the classes setting